### PR TITLE
chore: upgrades playwright to 1.45.3

### PR DESCRIPTION
**Pull request by zomars**
## What does this PR do?

Bumping version to address bug when using VS Code Playwright plugin on latest.

https://github.com/microsoft/playwright/issues/31936

## Mandatory Tasks (DO NOT REMOVE)

- [x] I have self-reviewed the code (A decent size PR without self-review might be rejected).
- [x] NA. I have added a Docs issue [here](https://github.com/calcom/docs/issues/new) if this PR makes changes that would require a [documentation change](https://docs.cal.com). If N/A, write N/A here and check the checkbox.
- [x] I confirm automated tests are in place that prove my fix is effective or that my feature works.

## How should this be tested?

- Are there environment variables that should be set?
- What are the minimal test data to have?
- What is expected (happy path) to have (input and output)?
- Any other important info that could help to test that PR

## Checklist

- I haven't read the [contributing guide](https://github.com/calcom/cal.com/blob/main/CONTRIBUTING.md)
- My code doesn't follow the style guidelines of this project
- I haven't commented my code, particularly in hard-to-understand areas
- I haven't checked if my changes generate no new warnings

### CODE REVIEW

File: apps/web/playwright/lib/fixtures.ts [hunk 1]
```diff
@@ -117,3 +119,8 @@ export const test = base.extend<Fixtures>({
     await use(webhooks);
   },
 });
+
+export function todo(title: string) {
+  // eslint-disable-next-line playwright/no-skipped-test
+  test.skip(title, noop);
+}
```
Comment (zomars) [inline apps/web/playwright/lib/fixtures.ts:126]: Moved here to fix new error:

```
Error: Can't call test() inside a describe() suite of a different test type.
Make sure to use the same "test" function (created by the test.extend() call) for all declarations inside a suite.
```

File: apps/web/playwright/impersonation.e2e.ts [hunk 2]
```diff
@@ -5,7 +5,7 @@ import { test } from "./lib/fixtures";
 test.describe.configure({ mode: "parallel" });
 
 test.describe("Users can impersonate", async () => {
-  test.afterAll(async ({ users }) => {
+  test.afterEach(async ({ users }) => {
```
Comment (zomars) [inline apps/web/playwright/impersonation.e2e.ts:8]: Seems like afterAll doesn't have fixtures any more

Comment (mxschmitt) [reply apps/web/playwright/impersonation.e2e.ts:8]: Before it was more a 'bug' that it had access to 'test fixtures'. 'test fixtures' -> fixtures which get created for every test, they should be used in {before,after}Each.

**Review by emrysal — APPROVED** [review]
Makes sense, LGTM, simple and small (once you see through the meaning)

**Pull request merged**